src/server/pfs/server/testing/load/specs/load-4.yaml

```diff
@@ -0,0 +1,36 @@
+count: 5
+operations:
+  - count: 5
+    fuzzOperations:
+      - operation:
+          putFile:
+            files:
+              count: 5
+              fuzzFile:
+                - file:
+                    source: "random"
+                  prob: 1
+        prob: 1
+validator: {}
+fileSources:
+  - name: "random"
+    random:
+      fuzzSize:
+        - size:
+            min: 1000
+            max: 10000
+          prob: 0.3
```

Comment:
A better way to do this is with weights. So you don't have to return an error if it doesn't sum to 1.

Reply:
I think probabilities are easier to understand for non-engineers (even engineers as well). I don't think it is a big deal that they need to sum to 1.